tests/Unit/Schema/Directives/DeprecatedDirectiveTest.php

```diff
@@ -58,7 +58,7 @@ enumValues(includeDeprecated: $includeDeprecated) {
 
         $types = $withoutDeprecatedIntrospection->json('data.__schema.types');
         $foo = Arr::first($types, static fn (array $type): bool => $type['name'] === 'Foo');
-        $this->assertCount(1, $foo['enumValues']);
```

Comment:
Alternatively, we could just ignore the error if you don't want the added noise in the test